addons/yatl/chat/chat.gd

```diff
@@ -1,163 +1,171 @@
-tool
-extends Node
-
-
 # Private imports
 
 const __PAL: Resource = preload("../pal/pal.gd")
+const __WebSocketConnection: Resource = __PAL.WebSocketConnection
```

Comment:
What's your though on preloading directly from `../shared/web_socket_connection.gd` instead?

Reply:
I don't care much for either, but one point for having it from PAL is that we can maintain a single point of access, in case we want to move the files.

Reply:
I do like that point in general, but in this case I think that as PAL isn't the owner of the shared type, it might not make the most sense. Unless we decide to move that into PAL full tile and have it accessed this way. Thoughts?